libcodechecker/server/database/database.py

```diff
@@ -376,7 +542,6 @@ def connect(self, db_version_info, init=False):
                     '-h', self.host,
                     '-p', str(self.port),
                     '-U', self.user,
-                    '-d', self.database,
```

Comment:
If I start the server like this:

`CodeChecker server --verbose debug --dbpost 6251 --dbuser whisperity --dbname codechecker_config_2`

(Where `codechecker_config_2` is an existing database which has just been created!), this happens:

```
[12:25:51] {DATABASE HANDLER} [4514] <139729816016640> - database.py:539 connect() - Checking if database is running at [localhost:6251]
[12:25:51] {UTIL} [4514] <139729816016640> - util.py:135 call_command() - Run psql -h localhost -p 6251 -U whisperity -c SELECT version();
[12:25:51] {UTIL} [4514] <139729816016640> - util.py:144 call_command() - Running command "psql -h localhost -p 6251 -U whisperity -c SELECT version();" Failed.
[12:25:51] {UTIL} [4514] <139729816016640> - util.py:145 call_command() - 2
[12:25:51] {UTIL} [4514] <139729816016640> - util.py:146 call_command() - psql: FATAL:  database "whisperity" does not exist
[12:25:51] {SERVER} - server.py:683 main() - Connection failed.
[12:25:51] {SERVER} [4514] <139729816016640> - server.py:730 main() - Status was not in the arguments.
[12:25:51] {SERVER} [4514] <139729816016640> - server.py:738 main() - Product upgrade was not in the arguments.
[12:25:51] {SERVER} [4514] <139729816016640> - server.py:437 check_product_db_status() - Checking config database
[... stack trace ...]
ProgrammingError: (psycopg2.ProgrammingError) relation "products" does not exist
LINE 2: FROM products
             ^
 [SQL: 'SELECT products.id AS products_id, products.endpoint AS products_endpoint, products.connection AS products_connection, products.display_name AS products_display_name, products.description AS products_description \nFROM products']
```